presidio-analyzer/presidio_analyzer/analyzer_engine_provider.py

```diff
@@ -0,0 +1,132 @@
+import yaml
```

Comment:
Not necessarily in the PR, but I would add two kind of tests - one for the unit suite covering this class, and one in the integration suite, reading different yamls from the filesystem, and seeing how they are processed by the class.